modules/liveIntentIdSystem.js

```diff
@@ -242,6 +242,59 @@ export const liveIntentIdSubmodule = {
     }
 
     return { callback: result };
+  },
+  eids: {
```

Comment:
@dgirardi I have a question related to changes in this PR: LiveIntent's user id module can also provide uid2s, but in case the `uid2idSystem` is not part of the Prebid built, it will be impossible to convert them into the eids. 

What should the intended usage pattern be in general? Is it "if you want an id of a certain type to be provided by a 'secondary' provider, you must have the 'main' provider too"? Or is it not the intention and LiveIntent's user id module should have a recipe for converting uid2s into eids?

Reply:
@peixunzhang FYI